src/python/grpcio_reflection/grpc_reflection/v1alpha/reflection.py

```diff
@@ -120,6 +118,7 @@ def _list_services(self):
             ]))
 
     def ServerReflectionInfo(self, request_iterator, context):
+        #pylint: disable=unused-argument
```

Comment:
Yes, *I guess*, it probably applies to the "function scope" since the warning applies to the function; but it did complain before I added the comment.